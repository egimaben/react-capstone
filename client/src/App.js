import React, { Component } from 'react'
import { connect } from 'react-redux'
import Routes from './routes/Routes'
import { Router } from 'react-router-dom'
import { createBrowserHistory } from "history";
const hist = createBrowserHistory();
export class App extends Component {

  render() {
    return (
        <Router history={hist}>
          <Routes />
        </Router>
    );
  }
}
export default App;
