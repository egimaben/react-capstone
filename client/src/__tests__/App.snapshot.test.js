import React from 'react';
import { App } from '../App';
import renderer from 'react-test-renderer'

console.error = err => { throw new Error(err); };
console.warn = warning => { throw new Error(warning); };

it('app renders correctly', () => {
  const fetchToken = jest.fn()
  const rendered = renderer.create(<App fetchToken={fetchToken} />);
  expect(rendered.toJSON()).toMatchSnapshot();
});