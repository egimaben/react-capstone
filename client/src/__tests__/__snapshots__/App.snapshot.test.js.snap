// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders correctly 1`] = `
<div
  id="wrapper"
>
  <div
    className="page-intro"
  >
    <a
      href="http://localhost:5000/api/auth/google"
    >
      <img
        alt="imio"
        className="sign-in-button"
        src="button.png"
      />
    </a>
  </div>
</div>
`;
